pkg/storage/store.go

```diff
@@ -170,9 +170,6 @@ func (s *store) init() error {
 		s.composite.AddStore(p.From.Time, f, idx, w, stop)
 	}
 
-	if s.cfg.EnableAsyncStore {
```

Comment:
can we get rid of the `EnableAsyncStore` property from the `Config` struct in `pkg/storage/factory.go` now? I think this was the only place it was being used.

Reply:
done!